src/platform_impl/linux/event_loop.rs

```diff
@@ -81,6 +81,8 @@ pub struct EventLoop<T: 'static> {
   events: crossbeam_channel::Receiver<Event<'static, T>>,
   /// Draw queue of EventLoop
   draws: crossbeam_channel::Receiver<WindowId>,
+  /// Textures
+  textures: crossbeam_channel::Receiver<Vec<u8>>,
```

Comment:
Could wrap into some generic Texture-struct?

Performance-wise, any challenges for using channels? There's probably only one item in queue anyway.